### ISSUE TITLE
Hide Lobby ID from URL

### ISSUE THREAD

**Issue by nawkht** [scribble-rs/scribble.rs#173]
For streamers, on twitch as an example, if I tried to play with my friends on stream, I should hide the URL with a trick on OBS so people watching the stream can't see the Lobby ID in the URL and flood the room/server.
Skribbl as an example have a box that you hover over it and then copy the invite link for you friends.
After joining the room, there is nothing showing on the top of the URL.
Adding something like this would be useful.
Thanks.

**Comment by Bios-Marcel** [comment]
This was a design decision, as I wanted having a "browser native" workflow. This way I can just quickly hit Ctrl + L and Ctrl + C and the lobby address is in my clipboard.

Another interesting scenario that doesn't work in skribbl.io, is that you can't hit F5 or Back and Forward again. My mouse for example has a "back" button which I hit by accident sometimes. How do I get back into a public lobby for example?

However, I see why this would be useful for a streamer. Would it be enough to be able to do this as a client-side setting? Meaning you'd have to opt-in. Or maybe you have a better idea to approach both of these problems?

**Comment by nawkht** [comment]
Yes true, I remember accidently hitting my MOUSE4 in skribbl.io couple of times and then I wasn't be able to get back to the game.
Hmmm, how about having a password for the lobby ?
Like you see the ( Public Lobby checkbox ) at the main page ?
Maybe like if it's not checked, it should be asking for a password.
If someone random tried  to enter the lobby, he shouldn't be able to play unless he have the right passwords.
What do you think about that approach ? You think it's doable ?

**Comment by Bios-Marcel** [comment]
Of course it is doable ^^

Funnily enough there was a password box in the beginning, but I thought it was useless, so I removed it.
Soooo, we'd have private / public + optional password. Should a public lobby still be able to have a password then?
I think it's kind of useless for a public lobby though, since lobbies don't have names.

The homepage is getting more and more messy as well, I was thinking that I might have redesign it either way.

**Comment by Bios-Marcel** [comment]
Well, no promises (Didn't work out well for me in the past). We'll see when I get around to do this.

**Comment by nawkht** [comment]
Well, it's not important. You don't have to waste your time on it. 
The password thingy should be enough.
And thanks for your time.

**Comment by nawkht** [comment]
This is unrelated, but i think maybe tou should add some donation links, i'm pretty sure many people would like to help.

**Comment by Bios-Marcel** [comment]
Here you go: https://github.com/scribble-rs/scribble.rs#donating

Thanks in case you plan to donate ^^
